src/groups/mqb/mqbblp/mqbblp_clusterqueuehelper.cpp

```diff
@@ -2127,6 +2127,10 @@ void ClusterQueueHelper::onHandleReleased(
 {
     // executed by the *QUEUE* dispatcher thread
 
+    // PRECONDITIONS
+    BSLS_ASSERT_SAFE(handle);
+    BSLS_ASSERT_SAFE(requester);
```

Comment:
These precondition checks are useful because they can catch an empty `handle` or `requester` in the original thread, before passing it to another thread's stack